app_customers/lib/widgets/custom_app_bar.dart

```diff
@@ -64,49 +65,64 @@ class _AppBarWidget extends StatelessWidget {
     final localization = Get.find<AppInternationalization>();
     final addThemeData = Get.find<ThemeManager>();
     final theme = Theme.of(context);
+    final customDrawerController = Get.find<CustomDrawerController>();
 
-    return Container(
-      color: theme.appBarTheme.backgroundColor,
-      height: 80,
-      width: MediaQuery.of(context).size.width,
-      child: Padding(
-        padding: const EdgeInsets.all(8.0),
-        child: Row(
-          mainAxisAlignment: MainAxisAlignment.center,
-          children: [
-            Expanded(
-              child: Align(
-                alignment: Alignment.center,
-                child: Text(
-                  localization.appName,
-                  style: TextStyle(
-                    fontWeight: FontWeight.bold,
-                    color: AppColors.white,
-                    fontSize: 19,
-                  ),
+    return Obx(
+      () => Container(
+        decoration: BoxDecoration(
+          borderRadius: BorderRadius.only(
+            topLeft: Radius.circular(customDrawerController.raduis.toDouble()),
+          ),
+          color: theme.appBarTheme.backgroundColor,
+        ),
+        height: 80,
+        width: MediaQuery.of(context).size.width,
+        child: Padding(
+          padding: const EdgeInsets.all(8.0),
+          child: Row(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              InkWell(
+                onTap: customDrawerController.showDrawer,
+                child: Icon(
```

Comment:
make remote config to hide this icon when drawer menu is disable.

Reply:
He doesn't need to create a remote config for that. He can solve it by using the remote with key 'displayDrawerMenuEnabled'.

Reply:
its ok